fri:
  port: 30200
  robot_ip: 192.170.10.2
  robot_description: /robot_description

hardware_interface:
  control_freq: 200 # in Hz
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
    - joint_7

publish:
  additional_info: true
  commanded_torques: true
